model/publishing/delivery/listeners/DeliveryEventsListeners.php

```diff
@@ -63,6 +64,7 @@ public static function updatedDeliveryEvent(DeliveryUpdatedEvent $event)
     {
         $delivery = new \core_kernel_classes_Resource($event->getDeliveryUri());
         try {
+            self::checkSyncProperty($delivery);
```

Comment:
In case if `DELIVERY_REMOTE_SYNC_FIELD` is not set error report will be returned.
This is not an error but expected behavior.